consensus/polybft/commitment.go

```diff
@@ -224,58 +119,41 @@ func (cm *CommitmentMessage) Hash() (types.Hash, error) {
 	commitment := map[string]interface{}{
 		"startId": cm.FromIndex,
 		"endId":   cm.ToIndex,
-		"leaves":  cm.BundlesCount(),
 		"root":    cm.MerkleRootHash,
 	}
 
-	data, err := bundleABIType.Encode(commitment)
+	data, err := commitmentABIType.Encode(commitment)
 	if err != nil {
 		return types.Hash{}, err
 	}
 
 	return crypto.Keccak256Hash(data), nil
 }
 
-// GetBundleIdxFromStateSyncEventIdx resolves bundle index based on given state sync event index
-func (cm *CommitmentMessage) GetBundleIdxFromStateSyncEventIdx(stateSyncEventIdx uint64) uint64 {
-	return (stateSyncEventIdx - cm.FromIndex) / cm.BundleSize
-}
-
-// GetFirstStateSyncIndexFromBundleIndex returns first state sync index based on bundle size and given bundle index
-// (offseted by FromIndex in CommitmentMessage)
-func (cm *CommitmentMessage) GetFirstStateSyncIndexFromBundleIndex(bundleIndex uint64) uint64 {
-	if bundleIndex == 0 {
-		return cm.FromIndex
-	}
-
-	return (cm.BundleSize * bundleIndex) + cm.FromIndex
-}
-
 // ContainsStateSync checks whether CommitmentMessage contains state sync event identified by index,
 // by comparing given state sync index with the bounds of CommitmentMessage
 func (cm *CommitmentMessage) ContainsStateSync(stateSyncIndex uint64) bool {
 	return stateSyncIndex >= cm.FromIndex && stateSyncIndex <= cm.ToIndex
 }
 
-// BundlesCount calculates bundles count contained in given CommitmentMessge
-func (cm *CommitmentMessage) BundlesCount() uint64 {
-	return (cm.ToIndex - cm.FromIndex + cm.BundleSize) / cm.BundleSize
+// StateSyncsCount calculates state syncs count contained in given CommitmentMessage
+func (cm *CommitmentMessage) StateSyncCount() uint64 {
+	return cm.ToIndex - cm.FromIndex
 }
 
-// VerifyProof validates given bundle proof against merkle trie root hash contained in the CommitmentMessage
-func (cm CommitmentMessage) VerifyProof(bundle *BundleProof) error {
-	if len(bundle.StateSyncs) == 0 {
-		return errors.New("no state sync events")
+// VerifyStateSyncProof validates given state sync proof
+// against merkle trie root hash contained in the CommitmentMessage
+func (cm CommitmentMessage) VerifyStateSyncProof(stateSyncProof *types.StateSyncProof) error {
```

Comment:
Only used in tests. What do you think, should we remove it, since now we don't have verification on client?
Maybe it's good to leave it in tests, like a way if we mess something up, should show us that something is wrong with state sync proofs?

Reply:
Agree